packages/suite/src/views/wallet/trading/common/TradingForm/TradingFormInput/TradingFormInputCurrency.tsx

```diff
@@ -65,10 +68,18 @@ export const TradingFormInputCurrency = ({
         }
     };
 
+    useEffect(() => {
```

Comment:
this should not be needed if `defaultCurrency` is set correctly right away (if possible, i dont know)